packages/core/adder/options.ts

```diff
@@ -137,12 +137,27 @@ export function prepareAndParseCliOptions<Args extends OptionDefinition>(adderDe
     const options = program.opts();
 
     if (multipleAdders) {
-        options.adder = program.args;
+        const selectedAdderIds: string[] = program.args ?? [];
```

Comment:
this should be inferable
```suggestion
        const selectedAdderIds = program.args ?? [];
```